### PR TITLE
Rename "All messages" to "Combined feed" in the codebase.

### PR DESCRIPTION
**Pull request by Lalit3716**
Fixes: #676

### CODE REVIEW

File: lib/model/narrow.dart [hunk 1]
```diff
@@ -39,13 +39,9 @@ sealed class SendableNarrow extends Narrow {
   MessageDestination get destination;
 }
 
-/// The narrow called "All messages" in the UI.
-///
-/// This does not literally mean all messages, or even all messages
-/// that the user has access to: in particular it excludes muted streams
-/// and topics.
```
Comment (Lalit3716) [inline lib/model/narrow.dart:46]: I am not sure whether I should remove this comment now or not?

Comment (chrisbobbe) [reply lib/model/narrow.dart:46]: I think we can replace it with something like:

```dart
/// All messages the user has access to, excluding unsubscribed streams
/// and muted streams and topics. See [PerAccountStore.isTopicVisible].
```

**Review by chrisbobbe — COMMENTED** [review]
Thanks! This is close to being ready.

I noticed that you included `Fixes: #676` in both commits. Please only include the `Fixes: #676` line in the one commit that completely fixes the issue; that is, the commit at which the "all messages" name no longer appears in the codebase. (If you want, you could include a `Fixes-partly: #676` in the first commit, but I think we don't have a consistent pattern of doing that.)

Also, I find one remaining reference to `AllMessagesNarrow`, in test/widgets/message_list_test.dart.

**Comment by chrisbobbe** [comment]
It looks like some conflicts have appeared. Please resolve those and comment when this is ready for another review.

File: lib/model/narrow.dart (hunk 1 above)
Comment (Lalit3716) [reply lib/model/narrow.dart:46]: Sounds good done!

**Comment by Lalit3716** [comment]
Resolved the conflicts and removed the `Fixes` from the first commit. This is ready for another look.

File: lib/model/narrow.dart [hunk 2]
```diff
@@ -39,13 +39,11 @@ sealed class SendableNarrow extends Narrow {
...
-class AllMessagesNarrow extends Narrow {
-  const AllMessagesNarrow();
+/// The narrow called "Combined feed" in the UI.
+/// All messages the user has access to, excluding unsubscribed streams
+/// and muted streams and topics. See [PerAccountStore.isTopicVisible].
```
Comment (chrisbobbe) [inline lib/model/narrow.dart:44]: nit: empty line between dartdoc's summary line and body

**Review by chrisbobbe — COMMENTED** [review]
Thanks! LGTM except one nit below, but I'll go ahead and label this for Greg to review.

**Review by gnprice — COMMENTED** [review]
Thanks @Lalit3716 and @chrisbobbe!

Apart from the comment Chris made above, I reran the search from the issue, and it looks like there's one match remaining that should be updated:
```
$ git grep -i 'all.?messages' 
…
lib/widgets/message_list.dart:            // widgets. As a simple test, flinging through All Messages in
```

**Comment by Lalit3716** [comment]
Updated ready for another look now!

**Comment by gnprice** [comment]
Thanks for the revision! Looks good — merging.

**Pull request merged**